scripts/styleguide.styles.scss

```diff
@@ -42,3 +42,9 @@
         height: 600px;
     }
 }
+
+.flex-center-center {
```

Comment:
I realize this is just the example styles, but ideally this should be a mixin rather than a class, or at least a utility class, like `.bdl-u-flexCenter`.